Formulario.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ejercicio Formulario</title>
+    <link rel="stylesheet" href="instalar%20bp-ac/css/bootstrap.min.css">
+    <script src="instalar%20bp-ac/js/bootstrap.bundle.js"></script>
+    <style>
+        body{
+            background-color: #badce3;
+        }
+    </style>
+
+</head>
+<body>
+<section>
+    <div class="text-black text-center ">
+        <p style="font-size: x-large">EJERCICIO FORMULARIO HTML</p>
+         </div>
+        <div class="container">
+        <form class="row g-3 mt-3">
+            <div class="col-6">
+                <label for="nombre" class="form-label">Nombre</label>
+                <input type="text" class="form-control" name="nombre" id="nombre" placeholder="  Ingrese su Nombre  ">
+            </div>
+            <div class="col-6">
+                <label for="apaterno" class="form-label">Apellido Paterno</label>
+                <input type="text" class="form-control" name="napaterno" id="apaterno" placeholder="  Ingrese su Apellido Paterno  ">
+            </div>
+            <div class="col-6">
+                <label for="amaterno" class="form-label">Apellido Materno</label>
+                <input type="text" class="form-control" name="amaterno" id="amaterno" placeholder="  Ingrese su Apellido Materno  ">
```

Comment:
evita los espacios en los valores de las propiedads de los nodos (excepto en la propeudad class ahi si hay espacios cuando hay mas de una clase)

```suggestion
   <input type="text" class="form-control" name="amaterno" id="amaterno" placeholder="Ingrese su Apellido Materno">
```